.github/workflows/tests_and_reports.yml

```diff
@@ -52,7 +49,7 @@ jobs:
         run: npm run test:unit:coverage
 
       - name: Report code coverage to PR in comment
-        if: github.ref != 'refs/heads/main'
+        if: ${{ !fromJSON(env.IS_MAIN) }}
```

Comment:
I was hoping to just run `!env.IS_MAIN` but the value of `IS_MAIN` is a string, not a boolean, so we need to use the `fromJSON` helper to cast it to a boolean. Not super happy with this, but it's probably still better than checking if `github.ref == 'refs/heads/main'` every time.